athena-gcs/athena-gcs.yaml

```diff
@@ -0,0 +1,118 @@
+Transform: 'AWS::Serverless-2016-10-31'
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: AthenaGcsConnector
+    Description: 'This connector enables Amazon Athena to communicate with Google Cloud Storage (GCS) and fetch data from Parquet file format'
+    Author: 'default author'
+    SpdxLicenseId: Apache-2.0
+    LicenseUrl: LICENSE.txt
+    ReadmeUrl: README.md
+    Labels: ['athena-federation', 'GCS', 'Google Cloud Storage', 'parquet', 'csv']
+    HomePageUrl: https://github.com/awslabs/aws-athena-query-federation
+    SemanticVersion: 2022.42.2
+    SourceCodeUrl: https://github.com/awslabs/aws-athena-query-federation
+
+Parameters:
+  AthenaCatalogName:
+    Description: 'The name you will give to this catalog is a Lambda name for Athena. Athena will use this name as the function name. This name must satisfy the pattern ^[a-z0-9-_]{1,64}$'
+    Type: String
+    AllowedPattern: ^[a-z0-9-_]{1,64}$
+  SpillBucket:
+    Description: 'The name of the bucket where this function can spill data.'
+    Type: String
+  SpillPrefix:
+    Description: 'The prefix within SpillBucket where this function can spill data.'
+    Type: String
+    Default: athena-spill
+  LambdaTimeout:
+    Description: "Maximum Lambda invocation runtime in seconds. (min 1 - 900 max)"
+    Default: 900
+    Type: Number
+  LambdaMemory:
+    Description: "Lambda memory in MB (min 128 - 3008 max)."
+    Default: 3008
+    Type: Number
+  DisableSpillEncryption:
+    Description: "WARNING: If set to 'true' encryption for spilled data is disabled."
+    Default: "false"
+    Type: String
+  VpcId:

```

Comment:
For google related connector which requires an internet connectivity. We can't set up the VPC right? we should remove this since this data source doesn't support VPC

Reply:
As per our understanding regarding VPC need for this connector, we can associate our lambda to a private subnet which is having the NAT gateway attached, this will enable internet access.

Reply:
>we can associate our lambda to a private subnet which is having the NAT gateway attached, this will enable internet access

That is correct, however, since this data source live outside of AWS and required internet access anyway. The value of VPC config is diminished to minimal. Therefore, i think we should remove VPC config for GCS to prevent confusion on customer

Reply:
yes, agree. we have already removed it.

Reply:
I checked athena-gcs.yaml. we still have vpc related config in athena-gcs.yaml.

```
VpcId
```

Reply:
taken care.